### PR TITLE
Enemy path 

### PR DESCRIPTION
**Pull request by stped18**
enemy part 
and som changes for Nicklas

### CODE REVIEW

File: dk.sdu.se4.common/src/main/java/dk/sdu/se4/common/entity/part/LifePart.java [hunk 1]
```diff
@@ -30,13 +30,13 @@ public void hit(int amount){
...
     public void addHitPoint(int amount){
         this.hit_point+=amount;
-        if (this.hit_point>this.MAX_HIT_POINTS){
-            this.hit_point=this.MAX_HIT_POINTS;
+        if (this.hit_point>this.max_hit_point){
+            this.hit_point=this.max_hit_point;
         }
     }
 
     public void setMAX_HIT_POINTS(int MAX_HIT_POINTS) {
```
Comment (Nicklason) [inline dk.sdu.se4.common/src/main/java/dk/sdu/se4/common/entity/part/LifePart.java:38]: Still not camel case :)

**Review by Nicklason — CHANGES_REQUESTED** [review]

File: dk.sdu.se4.common/src/main/java/dk/sdu/se4/common/entity/part/LifePart.java (hunk 1 above)
Comment (stped18) [reply dk.sdu.se4.common/src/main/java/dk/sdu/se4/common/entity/part/LifePart.java:38]: it is fixt

Comment (Nicklason) [reply dk.sdu.se4.common/src/main/java/dk/sdu/se4/common/entity/part/LifePart.java:38]: It is not camel case, it is snake case.

**Review by Nicklason — APPROVED** [review]

**Review by Kjalris — APPROVED** [review]

**Review by DenBedste — COMMENTED** [review]
Looks good

**Pull request merged**